### ISSUE TITLE
Geneared obj-c class name is not changing after project name is changed in podfile

### ISSUE THREAD

**Issue by zcsipler** [orta/cocoapods-keys#168]
When I started to use cocoapods-keys, for the first try I used the sample code to test, so I added Eidolon as project name. 

    plugin 'cocoapods-keys', {
        :project => "Eidolon",
        :keys => [
        "SomeTestKey"
        ]}

Then a class was generated, correctly named "EidolonKeys" . Later I wanted to use the real name of my project so I changed it in the podfile:

    plugin 'cocoapods-keys', {
        :project => "MyProject",
        :keys => [
        "SomeTestKey"
        ]}

After running either "pod install" or "pod update" still "EidolonKeys" class is generated instead of "MyProjectKeys". I would expect that the name of the generated class is always generated depending on the name which is defined in the podfile, in order to keep consistency with the ci and with other colleges who might be working on the same project. Am I doing something wrong, is it somehow possible to fix this issue?

**Comment by orta** [comment]
Try run `pod deintegrate` then `pod install` again, it shouldn't be keeping it around

**Comment by zcsipler** [comment]
Just tried ,it doesn't work, still EidolonKeys class is generated :(

**Comment by orta** [comment]
Is it possible it's doing both?

**Comment by zcsipler** [comment]
No, the only one class generated is "EidolonKeys", nothing else. And even if I update the podfile with a new key e.g.

plugin 'cocoapods-keys', {
    :project => "MyProject",
    :keys => [
    "SomeTestKey",
    "SomeOtherTestKey"
    ]}

the new property "someOtherTestKey" is inserted in EidolonKeys class. So it definitely updates, only the class name remains the old one

**Comment by orta** [comment]
Try use Target as well then, it lets you specify the name

> If you need to specify a different project name from the target name, use the key :target to specify it.

e.g. 

```
plugin 'cocoapods-keys', {
  :project => "MyProject",
  :target => "MyProject"
  :keys => [
    "SomeTestKey",
    "SomeOtherTestKey"
  ]}
```

**Comment by zcsipler** [comment]
I already tried it but didn't help either.

**Comment by dfeddad** [comment]
I had the same issue today, and found out that changing the project name in the Podfile is not enough for cocoapods-key to pickup the new name, you have to change the name in the YAML file generated by cocoapods-key.

Follow these steps to fix this problem:

- Locate the generated YAML file for your project (you can find it in this folder ```~/.cocoapods/keys```)

The content of the file should look like this:

```YAML
---
keys:
- Key1
- Key2
-....
path: [PATH_TO_YOUR_PROJECT_FOLDER]
name: [PROJECT_NAME]
```

- Update the ```[PROJECT_NAME]```

**Comment by jarrodparkes** [comment]
@dfeddad solution worked for me. so did removing the YAML file from `~/.cocoapods/keys`

**Comment by jarrodparkes** [comment]
would it be possible to allow for the customizing of the name of the obfuscated class?

**Comment by deda9** [comment]
Perfect, that was awesome, it worked like charm!
